store/views.py

```diff
@@ -62,21 +74,75 @@ def loanBookView(request):
     If yes, then set the message to 'success', otherwise 'failure'
     '''
     # START YOUR CODE HERE
-    book_id = None # get the book id from post data
-
+    
+    bid = request.body.decode("utf-8").split("=")[1]
+    Mbook=Book.objects.filter(id=bid)
+    book = BookCopy.objects.filter(book=Mbook[0], status=True)[0]
 
+    if book:
+        book.borrower = request.user
+        book.borrow_date = timezone.datetime.today().date()
+        book.status = False
+        book.save()
+        response_data['message'] = 'success'
+    else:
+        response_data['message'] = 'failure'
     return JsonResponse(response_data)
 
+
 '''
 FILL IN THE BELOW VIEW BY YOURSELF.
 This view will return the issued book.
 You need to accept the book id as argument from a post request.
 You additionally need to complete the returnBook function in the loaned_books.html file
 to make this feature complete
-''' 
+'''
+
+
 @csrf_exempt
```

Comment:
While `@csrf_exempt` would work here but it is considered a good practice to use it in application that are pure Django based (like this one) for security reasons. When sending requests using jquery ajax you can simply query the DOM to get hidden csrf element and fetch its value and include it in your request.